src/Symfony/Component/Workflow/MarkingStore/MarkingStoreInterface.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Workflow\MarkingStore;
+
+use Symfony\Component\Workflow\Marking;
+
+/**
+ * MarkingStoreInterface.
+ *
+ * @author Grégoire Pineau <lyrixx@lyrixx.info>
+ */
+interface MarkingStoreInterface
+{
+    public function getMarking($subject);
```

Comment:
@lyrixx Having phpdoc like:
```php
interface MarkingStoreInterface
{
    /**
     * @return Marking
     */
    public function getMarking($subject);
}
```
Allows IDE's to support this code easier, like highlight etc. cause if you declare that variable contains this interface (not class that implements it) highlight will not work.

Also this is _kinda_ part of Symfony CS, IIRC.